File: 양성주/2기/4주차/정수 내림차순으로 배치하기.java
import java.util.Arrays;

class Solution {
    public long solution(long n) {
        String answer = "";
		String str = n+"";
        char[] ch = str.toCharArray();
        Arrays.sort(ch);
        for(int i=ch.length-1;i>=0;i--) {
        	answer += ch[i];
        }
        long num = Long.parseLong(answer);
        return num;
    }
}

File: 이준혁/2기/15주차/정수 내림차순으로 배치하기.js
function solution(n) {
    let answer = 0;

    let tmp = [];

    while (true) {
        tmp.push(n % 10);
        n = n - (n % 10);
        n = n / 10;

        if (n === 0) {
            break;
        }
    }

    tmp.sort();

    for (let i = 0; i < tmp.length; i++) {
        answer += tmp[i] * 10 ** i;
    }
    return answer;
}
